### PR TITLE
Fix nullability for UserManager in AOSP build

### PR DESCRIPTION
**Pull request by grote**

### CODE REVIEW

File: app/src/main/java/com/stevesoltys/seedvault/metadata/MetadataManager.kt [hunk 1]
```diff
@@ -312,7 +312,7 @@ internal class MetadataManager(
     private fun getUserName(): String? {
         val perm = "android.permission.QUERY_USERS"
         return if (context.checkSelfPermission(perm) == PERMISSION_GRANTED) {
-            val userManager = context.getSystemService(UserManager::class.java)
+            val userManager = context.getSystemService(UserManager::class.java) ?: return false
```
Comment (t-m-w) [inline app/src/main/java/com/stevesoltys/seedvault/metadata/MetadataManager.kt]: Am I misunderstanding this, or is this trying to return `false` as a `String?`?

**Review by t-m-w — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/stevesoltys/seedvault/metadata/MetadataManager.kt (hunk 1 above)
Comment (grote) [reply app/src/main/java/com/stevesoltys/seedvault/metadata/MetadataManager.kt]: that was supposed to be `null`. sorry. I did this right from the web ui as I am working on another branch at the moment.

**Review by theimpulson — APPROVED** [review]
Looks good to me

**Comment by t-m-w** [comment]
Might be good to remove the first commit from this? Or will they get squashed automatically?

**Comment by chirayudesai** [comment]
> Might be good to remove the first commit from this? Or will they get squashed automatically?

We can squash from the UI when merging, which I did.

**Pull request merged**